eventmesh-common/src/main/java/org/apache/eventmesh/common/ThreadPoolFactory.java

```diff
@@ -41,13 +41,13 @@ public static ThreadPoolExecutor createThreadPoolExecutor(int core, int max, fin
 
     public static ThreadPoolExecutor createThreadPoolExecutor(int core, int max, BlockingQueue<Runnable> blockingQueue,
         final String threadName, final boolean isDaemon) {
-        return new ThreadPoolExecutor(core, max, 10 * 1000, TimeUnit.MILLISECONDS, blockingQueue,
+        return new LogThreadPoolExecutor(core, max, 10 * 1000, TimeUnit.MILLISECONDS, blockingQueue,
```

Comment:
These two `createThreadPoolExecutor` methods are used by `EventMeshGrpcServer` and `HTTPThreadPoolGroup`. Why their return type should be changed to `LogThreadPoolExecutor`?

Reply:
Returning the LogThreadPoolExecutor  can lay the groundwork for subsequent monitoring.

Reply:
May you please describe the subsequent monitoring features to be implemented?